analyzer/uncategorized/evm_raw_tx_test.go

```diff
@@ -60,8 +60,7 @@ func decodeExpectCall(
 	require.Equal(t, expectedDataBytes, body.Data)
 	require.Len(t, tx.AuthInfo.SignerInfo, 1)
 	from0xChecksummed := helpers.EthAddressFromPubKey(*tx.AuthInfo.SignerInfo[0].AddressSpec.Signature.Secp256k1Eth)
-	fromChecksummed := from0xChecksummed[2:]
-	from := strings.ToLower(fromChecksummed)
```

Comment:
`EthAddressFromPubKey` used to return a `string` (with a leading `0x`, but now returns `ethCommon.Address`, which is `[20]byte`.
I checked, and the rest of indexer is not using `EthAddressFromPubKey`, or even the `helpers` package.

Reply:
I gotta rename from0xChecksummend now :skull: